cerberus/validator.py

```diff
@@ -1240,6 +1241,14 @@ def _validate_type_integer(self, value):
         if isinstance(value, _int_types):
             return True
 
+    def _validate_type_decimal(self, value):
+        if isinstance(value, decimal.Decimal):
```

Comment:
i'd prefer [this](https://github.com/pyeve/cerberus/pull/332#issuecomment-323454495).